initialize/services.go

```diff
@@ -10,17 +10,17 @@ import (
 	"time"
 
 	"github.com/go-jose/go-jose/v3/jwt"
-	"github.com/jackc/pgx/v4/pgxpool"
-	"github.com/quay/clair/config"
+	"github.com/jackc/pgx/v5/pgxpool"
 	"github.com/quay/claircore/datastore/postgres"
 	"github.com/quay/claircore/enricher/cvss"
 	"github.com/quay/claircore/libindex"
 	"github.com/quay/claircore/libvuln"
 	"github.com/quay/claircore/libvuln/driver"
-	"github.com/quay/claircore/pkg/ctxlock"
+	"github.com/quay/claircore/pkg/ctxlock/v2"
 	"github.com/quay/zlog"
 	"golang.org/x/net/publicsuffix"
 
+	"github.com/quay/clair/config"
```

Comment:
nit: I figured this third section is for anything within the same module. This is in the same repo, but a different module. Up to you